superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, name, parameters=''):
                 'int4_tc': 'cutlass_tensorop_s4_i16864gemm_s4_256x128_128x3_*',
             }
         }
+        self.__kernel_map[7.5] = {k: self.__kernel_map[7.0][k] for k in self.__kernel_map[7.0] if 'fp64' not in k}
```

Comment:
Devices for inference don't  support fp64 kernels?

Reply:
Yes. @abuccts should write a comment for this.

Reply:
[Reference](https://images.nvidia.com/aem-dam/en-zz/Solutions/geforce/ampere/pdf/NVIDIA-ampere-GA102-GPU-Architecture-Whitepaper-V1.pdf) for A10,
> The GA102 GPU also features 168 FP64 units (two per SM), which are not depicted in this
diagram. The FP64 TFLOP rate is 1/64th the TFLOP rate of FP32 operations. The small number of FP64
hardware units are included to ensure any programs with FP64 code operate correctly

Reply:
Great!